iroh-willow/src/net.rs

```diff
@@ -30,6 +31,9 @@ pub const CHANNEL_CAP: usize = 1024 * 64;
 /// The ALPN protocol name for iroh-willow.
 pub const ALPN: &[u8] = b"iroh-willow/0";
 
+/// QUIC application error code for closing with failure.
+pub const ERROR_CODE_FAIL: VarInt = VarInt::from_u32(0);
```

Comment:
i usually reserve `0` for "accidentally dropped" because it used by default if you drop in quinn